### ISSUE TITLE
Issue with resolving dependencies

### ISSUE THREAD

**Issue by IgorKarpiuk** [Redocly/redocly-cli#1240]
**Describe the bug**

When `redoc` and `redocly/cli` are installed in the same repository, the `build-docs` command will not work because `npm` installs two versions of `react`

**To Reproduce**

1. Create new  project
2. Install `redoc` and `redocly-cli` via `npm`
3. Run `build-docs` command 

**Expected behavior**

The `build-docs` command should work

**Logs**

![Screenshot 2023-08-21 at 11 12 04](https://github.com/Redocly/redocly-cli/assets/106662428/6b919667-8535-4377-ab14-2560abd89a3c)

**Additional context**

![Screenshot 2023-08-21 at 11 13 40](https://github.com/Redocly/redocly-cli/assets/106662428/594fbb2b-f6c5-4f9a-be91-c26deea0ef3c)

**Comment by IgorKarpiuk** [comment]
Hi @akindo Could you please share the results of command `npm ls react` ?

**Comment by akindo** [comment]
```
❯ npm ls react
example@0.0.1 /Users/example/example
└─┬ @apps/notifications@0.0.1 -> ./apps/notifications
  ├─┬ @packages/notifications-utils@0.0.1 -> ./packages/notifications-utils
  │ └─┬ @packages/config-eslint@1.0.0 -> ./packages/config-eslint
  │   └─┬ eslint-config-next@12.1.0
  │     └─┬ next@12.3.1
  │       ├── react@18.2.0 deduped
  │       ├─┬ styled-jsx@5.0.7
  │       │ └── react@18.2.0 deduped
  │       └─┬ use-sync-external-store@1.2.0
  │         └── react@18.2.0 deduped
  └─┬ @redocly/cli@1.0.2
    ├─┬ react-dom@17.0.2
    │ └── react@17.0.2 deduped
    ├── react@17.0.2
    ├─┬ redoc@2.0.0
    │ ├─┬ mobx-react@7.6.0
    │ │ ├─┬ mobx-react-lite@3.4.3
    │ │ │ └── react@18.2.0 deduped
    │ │ └── react@18.2.0 deduped
    │ ├─┬ react-tabs@3.2.3
    │ │ └── react@17.0.2 deduped
    │ └── react@17.0.2 deduped
    └─┬ styled-components@5.3.11
      ├─┬ react-dom@18.2.0
      │ └── react@18.2.0 deduped
      └── react@18.2.0
```

**Comment by rpohodych** [comment]
Hello @IgorKarpiuk , I getting the same error:
<img width="1201" alt="image" src="https://github.com/Redocly/redocly-cli/assets/87315268/f58814a6-34cd-4e19-82a8-265d9f2cdea2">
Results of command `npm ls react`
<img width="432" alt="image" src="https://github.com/Redocly/redocly-cli/assets/87315268/4afaafe7-54b2-43ba-9395-6e3441fbc5ba">

**Comment by IgorKarpiuk** [comment]
Hi @akindo @rpohodych should be fixed in `v1.1.0`
